lib/user_space/user_x86_64_atomic.h

```diff
@@ -64,9 +69,13 @@ static inline int64_t m0_atomic64_get(const struct m0_atomic64 *a)
  */
 static inline void m0_atomic64_inc(struct m0_atomic64 *a)
 {
+#ifdef CONFIG_X86_64
 	asm volatile("lock incq %0"
 		     : "=m" (a->a_value)
 		     : "m" (a->a_value));
+#elif defined CONFIG_AARCH64 /* aarch64 */
```

Comment:
Actually, from the `CONFIG_AARCH64` it's obvious that it's about `aarch64`, so the comment is redundant here. (And in other similar places.) It only adds an excess noise.

Reply:
It is ok  to carry the comments.   it does not hinder any code assumption of understanding

Reply:
As I said - it's an excess noise which distracts. Drop it everywhere.